packages/lexical/src/LexicalEditor.ts

```diff
@@ -311,6 +312,7 @@ export function createEditor(editorConfig?: {
     RootNode,
     TextNode,
     LineBreakNode,
+    CodeLineNode,
```

Comment:
We should not add line node here (by default), it should be added by editor user if they actually want to have code nodes. IT should be registered at the same place where CodeNOde and CodeHighlightNode are registered